decidim-core/app/services/decidim/contextual_help_repository.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This class holds the logic to retrieve and set contextual help content
+  # that will be shown in different sections of the page.
+  class ContextualHelpRepository
```

Comment:
Why do we need this class instead of adding logic to `HelpElement`? We haven't used this pattern anywhere in the app.

Reply:
Actually, it ended up being quite small, so I think I'll move this as class methods to `HelpElement` (and rename it).

Reply:
But why do we need this class?